src/components/FAQ-page/Question.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Fab from '@material-ui/core/Fab';
+import AddIcon from '@material-ui/icons/Add';
+import './faq-style.css';
+
+
+class Question extends Component {
+    render() {
+      return (
+        <div >
+             <h2>Will a beginner, with absolutely no knowledge of github, gain anything fruitful? &nbsp;&nbsp;
+                 &nbsp;&nbsp;
+                <Fab size="small" color="secondary" aria-label="Add" onClick="myFunction()">
+                <AddIcon />
+                </Fab>
+                <br />
+                 <h6 id="myDIV">Yeah,definitely. The organization is meant to assist the beginners grow in the field of development. We’ll have distinct projects appropriate both for beginners as well as the accolades and thereby we’ll make sure that each and every participant gets to learn something new from the projects he or she is contributing for.
+                 </h6>
+                <hr />
+             </h2>
+             
+        </div>
+      );
+    }
+  }
+  export default Question;  
```

Comment:
Do you think this look better:
```
class Question extends Component {
  render() {
    return (
      <Grid container className={styles.root} spacing={16} wrap>
        <Grid item xs={12} wrap>
          <Typography paragraph className={styles.aboutText}>{this.props.questionText}</Typography>
          <Typography paragraph className={styles.aboutText}>{this.props.answerText}</Typography>
          <hr />
        </Grid>
      </Grid>
    );
  }
}
```

You have not used any material design components and used basic HTML and CSS. When building with material components, it's important to only use material components :)

Reply:
I have made all the changes which u suggested me except using material ui . Actually I dont have that much knowledge of  material-ui and u said just by using basic html css we can write code in react . So , this is my first attempt to write code in react and my basic motive to write code in react successfully rather than giving much attention to material ui . Can I use that in further pages  except this ?

Reply:
I have already provided you all the Material UI related code to you. All you have to do is copy paste it in your code to make it easy for you. All you have to do is make minor styling adjustments, that's it.

Sure, the HTML code you wrote works, but that is not the right way to do it, and writing HTML and CSS in react is not react development.

> I have made all the changes which u suggested me except using material ui

Have you pushed your changes? I don't see it.

> this is my first attempt to write code in react and my basic motive to write code in react successfully rather than giving much attention to material ui

I can clearly see you didn't read react.js documentation and material-ui documentation. 4 week is pretty long time to at least get the basics 😅

Reply:
what's the styles in it ?